src/deliveryrouting/src/GetPathComponent.css

```diff
@@ -0,0 +1,41 @@
+.ButtonContainer {
+  align-items: center;
+  display: flex;
+  height: 100%;
+  justify-content: center;   
+}
+
+.PreviousPathButton, .NextPathButton {
+  background-color: blue;
+  border: 1px solid #dadce0;
+  border-radius: 36px;
+  box-shadow: 0 1px 6px rgba(60, 64, 67, 0.28);
+  cursor: pointer;
+  font-family: "Google Sans", "Roboto", "Noto Sans Devanagari UI",
+    "Noto Sans Bengali UI", "Noto Sans Telugu UI", "Noto Sans Tamil UI",
+    "Noto Sans Gujarati UI", "Noto Sans Kannada UI", "Noto Sans Malayalam UI",
+    "Noto Sans Gurmukhi UI", "Arial", "sans-serif";
+  font-size: 15px;
+  font-weight: bold;
+  height: 30px;
+  margin-top: 20px;
+  outline: none;
+  width: 200px;
+}
+  
+.PreviousPathButton:hover, .NextPathButton:hover {
+  background-color: #4285f4;
+}
+
+.PreviousPathButton:disabled, .NextPathButton:disabled {
+  background-color: darkgrey;
+}
+
+.PathInformation {
+  font-family: "Google Sans", "Roboto", "Noto Sans Devanagari UI",
```

Comment:
Are these font open source? What license is being used?

Reply:
They are under Apache license, version 2 and free for use. But I realized their usage wasn't adding any value to the page, so I removed them.